npm/oxc-parser/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "oxc-parser",
   "version": "0.56.5",
   "main": "index.js",
-  "browser": "browser.js",
+  "browser": "wasm.js",
```

Comment:
Also purist in me wants to call this file `wasm.mjs` (though `browser` field is not standard either) :sweat_smile: